src/EPR.RegulatorService.Facade.API/Controllers/OrganisationsSearchController.cs

```diff
@@ -132,7 +140,9 @@ public async Task<IActionResult> GetUsersByOrganisationExternalId(Guid externalI
         }
         catch (Exception e)
         {
-            _logger.LogError(e,$"Error fetching producer organisations by external organisation id {externalId}");
+            string logData = string.Format($"Error fetching producer organisations by external organisation id {0}", externalId);
+            _logger.LogError(e, logData);
```

Comment:
use the logger and remove string
_logger.LogError(e, $"Error fetching producer organisations by external organisation id {0}", externalId);